package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@carletonuniversity/rds",
   "private": false,
-  "version": "0.1.2",
+  "version": "0.1.3",
   "license": "MIT",
```

Comment:
did you do a release ? we geenrally do this when we are doing a release .

Reply:
I reverted the version numbers back.